src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseTransferService.java

```diff
@@ -166,8 +140,60 @@ private boolean checkHearingsNotListed(CaseData caseData) {
                 }
             }
         }
+
         return true;
     }
 
+    private void validateCase(CaseData caseData, List<String> errors) {
+        if (!checkBfActionsCleared(caseData)) {
+            errors.add(String.format(BF_ACTIONS_ERROR_MSG, caseData.getEthosCaseReference()));
+        }
+
+        if (!checkHearingsNotListed(caseData)) {
+            errors.add(String.format(HEARINGS_ERROR_MSG, caseData.getEthosCaseReference()));
+        }
+    }
+
+    private List<String> transferCases(CaseDetails caseDetails, List<CaseData> caseDataList, String userToken) {
+        var caseTypeId = caseDetails.getCaseTypeId();
+        var officeCT = caseDetails.getCaseData().getOfficeCT().getValue().getCode();
+        var positionTypeCT = caseDetails.getCaseData().getPositionTypeCT();
+        var reasonForCT = caseDetails.getCaseData().getReasonForCT();
+        var jurisdiction = caseDetails.getJurisdiction();
+        var transferEventParams = new TransferEventParams(userToken, caseTypeId, officeCT, positionTypeCT, reasonForCT,
+                jurisdiction);
+
+        var errors = new ArrayList<String>();
+        for (CaseData caseData : caseDataList) {
+            createCaseTransferEvent(transferEventParams, caseData, errors);
```

Comment:
why not pass just caseDetails as parameter rather than caseData and transferEventParams to method createCaseTransferEvent?

Reply:
It's a good point. TransferEventParams removed